src/api/app/controllers/comments_controller.rb

```diff
@@ -23,15 +22,18 @@ def destroy
   def find_obj
     if params[:project]
       if params[:package]
-        @template = 'package'
         @obj = Package.get_by_project_and_name(params[:project], params[:package])
+        @header = { project: @obj.project.name, package: @obj.name }
```

Comment:
Yes, but I'm coming from the parent, the controller needs also to be polymorphic.